contracts/factories/WrappedExternalBribeFactory.sol

```diff
@@ -21,4 +18,10 @@ contract WrappedExternalBribeFactory {
         oldBribeToNew[existing_bribe] = last_bribe;
         return last_bribe;
     }
+
+    function setVoter(address _voter) external {
+        require(!_initialized, "Already initialized");
```

Comment:
don't need this flag? can just check voter is 0?

Reply:
removed. check with address 0